modules/ossm-migrating-complete-remove-2-6-operator-crds.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/done/ossm-migrating-complete-assembly
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-complete-remove-2-6-operator-crds_{context}"]
+= Remove the Service Mesh 2.6 operator and custom resource definitions
+
+Once you have migrated your deployment and workloads, you can remove the {SMproduct} 2.6 operator and custom resource definitions (CRDs).
+
+.Prerequisites
+
+* You have completed your deployment and workloads migrations.
+* You are logged in to the {ocp-product-title} web console as a user with the cluster-admin role.
```

Comment:
One more prereq is to finish procedure above "Remove the Service Mesh 2.6 control plane"